project/Dependencies.scala

```diff
@@ -18,7 +18,8 @@ object Dependencies {
 
   val workbenchUtilV = "0.5-4c7acd5"
   val workbenchModelV = "0.13-31cacc4"
-  val workbenchGoogleV = "0.21-96ad43c"
+  // TODO: revert to non-SNAP and remove this comment once https://github.com/broadinstitute/workbench-libs/pull/307 is merged
```

Comment:
think https://github.com/broadinstitute/workbench-libs/pull/307 can be merged?

Reply:
Yeah will do.. was just waiting for auto tests to pass